src/components/containers/show.js

```diff
@@ -24,24 +25,49 @@ const Location = ({ children, to }) => {
         if (!to) return
 
         const clients_country = Cookies.get('clients_country')
-        const showEu = handleEu(setVisible, to)
-        showEu(isEuCountry(clients_country))
-    })
+
+        if (clients_country) {
+            const showEu = handleEu(setVisible, to)
```

Comment:
only need one `showEu`, can move this declaration up one step and remove the one below